app/gui2/src/stores/visualization/compilerMessaging.ts

```diff
@@ -34,10 +36,19 @@ export const stylePathAttribute = 'data-style-path'
 
 const VisualizationModule = z.object({
   // This is UNSAFE, but unavoiable as the type of `Visualization` is too difficult to statically
-  // check. Insteead it will be caught by Vue when trying to mount the visualization, and replaced
+  // check. Instead it will be caught by Vue when trying to mount the visualization, and replaced
   // with a 'Loading Error' visualization.
   default: z.custom<Visualization>(() => true),
   name: z.string(),
+  // The name of an icon, or a URL or data URL. If it contains `:`, it is assumed to be a URL.
```

Comment:
This URL detection logic seems to not be implemented.

Reply:
ah... nice catch, thanks :sweat_smile: